### PR TITLE
Projects pagination

### LINKED ISSUE #329: Add pagination of projects
API supports it already, need to implement in front-end.

### PR DESCRIPTION
**Pull request by alanmoo**
Closes #329

### CODE REVIEW

File: app/react/pages/projects/projects.jsx [hunk 1]
```diff
@@ -14,34 +14,42 @@ export default React.createClass({
...
         sort: this.state.sortBy,
         categories: this.state.category,
-        expand: `leads`
+        expand: `leads`,
+        page: page
       })
-      .then((data) => { this.setState({projects: data.results}); })
+      .then((data) => {
+        this.setState({
+          projects: page === 1 ? data.results : this.state.projects.concat(data.results),
```
Comment (gvn) [inline app/react/pages/projects/projects.jsx]: Couldn't this just be:

`projects: this.state.projects.concat(data.results),`

**Comment by gvn** [comment]
1 nit & R+

File: app/react/pages/projects/projects.jsx (hunk 1 above)
Comment (alanmoo) [inline app/react/pages/projects/projects.jsx]: No, if the filter changes we don't want to concat the results.

Comment (gvn) [inline app/react/pages/projects/projects.jsx]: Ah got it

**Comment by alanmoo** [comment]
Added a comment to clarify that line

**Pull request merged**